javascript/javascript1/week3/smart-ease.js

```diff
@@ -0,0 +1,153 @@
+//Item array removal
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+
+const nameToRemove = "Ahmad";
+names.splice(1,1);  
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+
+//When will we be there??
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+function getThereFaster() {
+    //calculation of the required speed to arrive the destination in 3 hours and 34 minutes
+    const speficySpeed =(travelInformation.destinationDistance/(3+(34/60)));
+    travelInformation.speed =speficySpeed; 
+
+    //calculation of time
+    hours = (travelInformation.destinationDistance /travelInformation.speed);
+    onlyHours = Math.floor(hours);
+    minutes = (hours - onlyHours) * 60;
+    onlyMinutes = Math.round(minutes);
```

Comment:
You can't define a variable without using `let` or `const`. Which would you use here?